Share JSP 2.3, 3.0, & 3.1 org.apache.jasper & factory code (#26397)
OpenLiberty/open-liberty: dev/com.ibm.ws.jsp.2.3/README.txt

This bundle (after transformation) is the base for all JSP/Pages factory bundles used by later features, such as page-3.0, pages-3.1, and pages-4.0.

> Any changes applied here will also be picked up by the later bundles. If that is not indented, then overlay the original file in teh other projects( ie. io.openliberty.pages.3.1.internal.factories, io.openliberty.pages.4.0.internal.factories, ... etc)

Author: ```suggestion
Any changes applied here will also be picked up by the later bundles (all later features). If that is not the intent, then overlay the original file in the other projects ( i.e. io.openliberty.pages.3.1.internal.factories, io.openliberty.pages.4.0.internal.factories, ... etc)
```
